src/components/gauge/gauge.e2e.ts

```diff
@@ -0,0 +1,156 @@
+    await page.waitForChanges();
+    expect(await element.find("svg")).toBeTruthy();
+  });
+
+  /////////////////// Values test ///////////////////////
+  it("should set min value", async () => {
+    await page.setContent(
+      "<gx-gauge gauge-type='Circle' min-value='0'></gx-gauge>"
+    );
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    expect(await element.find("svg")).toBeTruthy();
+    expect(await element.getProperty("minValue")).toEqual(0);
+  });
+
+  it("should set current value", async () => {
+    await page.setContent(
+      "<gx-gauge gauge-type='Circle' min-value='0' current-value='25'></gx-gauge>"
+    );
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    expect(await element.find("svg")).toBeTruthy();
+    expect(await element.getProperty("minValue")).toEqual(0);
+    expect(await element.getProperty("currentValue")).toEqual(25);
+  });
+
+  it("should show the current value", async () => {
+    await page.setContent(
+      "<gx-gauge gauge-type='Line' min-value='0' current-value='25' show-value='true'><gx-gauge-range color='rgba(116, 16, 216, 1)' amount='100' name='Violet' class='hydrated'></gx-gauge-range></gx-gauge>"
+    );
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    const marker = await element.find("span.marker");
+    await page.waitForChanges();
+    expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+    expect(await element.getProperty("minValue")).toEqual(0);
+    expect(await element.getProperty("currentValue")).toEqual(25);
+    expect(await marker.getAttribute("style")).toEqual(
+      "box-shadow: none; margin-left: 24.9%;"
+    );
+  });
+
+  /////////////////// Frontend test ///////////////////////
+  it("should set thickness value", async () => {
+    await page.setContent(
+      "<gx-gauge gauge-type='Circle' min-value='0' current-value='25' thickness='20'></gx-gauge>"
+    );
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    expect(await element.find("svg")).toBeTruthy();
+    expect(await element.getProperty("thickness")).toEqual(20);
+  });
+
+  it("should set a marker in Line", async () => {
+    await page.setContent(
+      "<gx-gauge gauge-type='Line' min-value='0' current-value='25' show-value='true' style-shadow='false'><gx-gauge-range color='rgba(116, 16, 216, 1)' amount='100' name='Violet' class='hydrated'></gx-gauge-range></gx-gauge>"
+    );
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    await page.waitForChanges();
+    expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+    expect(await element.find("span.marker")).toBeTruthy();
+    expect(await element.find(".minMaxDisplay")).toBeTruthy();
```

Comment:
I would like to see an expectation for the `Violet` text.

Reply:
Test added in commit e372c6e70aaaae9d521022815fbda83e2af2fe45